git/cmd.py

```diff
@@ -13,6 +13,8 @@
 import errno
 import mmap
 
+from collections import OrderedDict
```

Comment:
Here is the result: https://travis-ci.org/gitpython-developers/GitPython/jobs/133041770 .

Seems to be just one part of the compat code, which may be fixable in some way in case the blame functionality is important to someone.